src/scripts.js

```diff
@@ -2,21 +2,33 @@
 // query selectors and event listeners in here 
 
 import './styles.css'
-import { renderGrid } from './domUpdates'
+import { renderGrid, showRecipe, closeRecipe } from './domUpdates'
 
 // import apiCalls from './apiCalls'
 
-const recipeGrid = document.querySelector('.recipe-grid')
-
+const recipeGrid = document.querySelector('.recipe-grid');
+const allRecipes = document.querySelector('.all-recipes');
+const clickedRecipe = document.querySelector('#clickedRecipe');
+const closeRecipeButton = document.querySelector('#closeRecipe');
 
 // // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 // import './images/turing-logo.png'
 // import ingredientsData from './data/ingredients.js'
 
 // //Example of one way to import functions from the domUpdates file. You will delete these examples.
 // import {exampleFunction1, exampleFunction2} from './domUpdates.js'
-window.addEventListener("load", renderGrid)
+window.addEventListener("load", renderGrid);
+
+allRecipes.addEventListener("click", (event) => {
+  if (event.target.classList.contains('individual-recipe')) {
```

Comment:
This is working beautifully so far. But JUST in case, if our event.target did not have a classList, we'd fact an error. 
If you make it 
if (event.target.classList?.contains) - interpretor would not continue with the rest of the code if event.target had no classlist.

Reply:
ahh interesting, i'll experiment with that!